tests/functional/TestSmokeTest/tests/heketi_test.go

```diff
@@ -465,6 +465,37 @@ func TestHeketiVolumeCreateWithOptions(t *testing.T) {
 
 }
 
+func TestHeketiVolumeCreateSetsIdOption(t *testing.T) {
+	na := testutils.RequireNodeAccess(t)
+	// Setup the VM storage topology
+	teardownCluster(t)
+	setupCluster(t, 2, 2)
+	defer teardownCluster(t)
+
+	// Create a volume
+	volReq := &api.VolumeCreateRequest{}
+	volReq.Size = 10
+	volReq.Durability.Type = api.DurabilityReplicate
+	volReq.Durability.Replicate.Replica = 2
+	volReq.Snapshot.Enable = true
+	volReq.Snapshot.Factor = 1.5
+
+	// Set to the vagrant gid
+	volReq.Gid = 2333
+
+	// Create the volume
+	volInfo, err := heketi.VolumeCreate(volReq)
+	tests.Assert(t, err == nil, "expected err == nil, got:", err)
+
+	// SSH into system and check for user.heketi.id option
+	vagrantexec := na.Use(logger)
```

Comment:
I'd like to use a more general name for the "exec" variable. Thanks.

Reply:
Not sure I understand, that’s copy and paste from a similar case, so at
least it’s consistent. Can you suggest a name?

Reply:
Yes, but this is a new test so I don't want to propagate it more. How about just `exec`?

Reply:
done